streams/src/test/java/org/apache/kafka/streams/state/internals/MeteredTimestampedWindowStoreTest.java

```diff
@@ -176,38 +183,34 @@ private void doShouldPassChangelogTopicNameToStateStoreSerde(final String topic)
             keySerde,
             valueSerde
         );
-        store.init((StateStoreContext) context, store);
 
+        store.init((StateStoreContext) context, store);
         store.fetch(KEY, TIMESTAMP);
         store.put(KEY, VALUE_AND_TIMESTAMP, TIMESTAMP);
 
-        verify(keySerializer, valueDeserializer, valueSerializer);
+        verify(innerStoreMock).fetch(any(Bytes.class), eq(TIMESTAMP));
+        verify(innerStoreMock).put(any(Bytes.class), any(byte[].class), eq(TIMESTAMP));
```

Comment:
We can also make this `verify()` call more specific by updating it to: `        verify(innerStoreMock).put(KEY_BYTES, VALUE_AND_TIMESTAMP_BYTES, TIMESTAMP);
`

Reply:
Ditto